src/permitInfo/utils/getProvider.ts

```diff
@@ -1,18 +1,14 @@
 import { JsonRpcProvider } from '@ethersproject/providers'
-import { DEFAULT_RPC_URLS } from '../const.ts'
-import { env } from 'node:process'
+import { DEFAULT_RPC_URLS } from '../const'
 import { ethers } from 'ethers'
+import {SupportedChainId} from '@cowprotocol/cow-sdk'
 
 export function getProvider(chainId: number, rpcUrl: string | undefined): JsonRpcProvider {
-  const rpcEndpoint = rpcUrl ? rpcUrl : DEFAULT_RPC_URLS[chainId]
+  const rpcEndpoint = rpcUrl ? rpcUrl : DEFAULT_RPC_URLS[chainId as SupportedChainId]
```

Comment:
Probably is nice to have eventually a `isSupportedChainId(chainId): chainId is SupportedChainId` so we don't need to cast like this

Reply:
Anyway, we have to cast it.
For example, when we get a value from a command line input it always will be a number

Reply:
yes, my point is more that if we have this, we don't need to cast and for example in the case of a CLI input, you can do:

if (!isSupportedChainId(chainId)) {
   console.error('Wrong chain', chainId)
}

// Here we already have casted chainID, we don't need `as`